app/core/models.py

```diff
@@ -11,14 +11,12 @@
 
 class AbstractBaseModel(models.Model):
     """
-    Base abstract model, that has `uuid` instead of `id` and included `created_at`, `updated_at` fields.
+    Base abstract model, that has `uuid` instead of `id` fields.
     """
 
     uuid = models.UUIDField(
         primary_key=True, default=uuid.uuid4, editable=False, unique=True
     )
-    created_at = models.DateTimeField("Created at", auto_now_add=True)
```

Comment:
why did we remove these fields? these are used by many other models. This looks related to the snapshot that I took in the max_migration.txt comment.